packages/components/organisms/f-checkout/src/components/_tests/Checkout.test.js

```diff
@@ -501,6 +543,106 @@ describe('Checkout', () => {
                 expect(result).toBe(false);
             });
         });
+
+        describe('shouldShowErrorDialog ::', () => {
+            it('should return `true` if `nonFulfillableError.shouldShowInDialog` is `true`', () => {
+                // Arrange
+                const wrapper = shallowMount(VueCheckout, {
+                    store: createStore({
+                        ...defaultCheckoutState,
+                        isLoggedIn: true
+                    }),
+                    i18n,
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            nonFulfillableError: {
+                                code: ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE,
+                                shouldShowInDialog: true
+                            }
+                        };
+                    }
+                });
+
+                // Act
+                const result = wrapper.vm.shouldShowErrorDialog;
+
+                // Assert
+                expect(result).toBe(true);
+            });
+
+            it('should return `false` if `nonFulfillableError.shouldShowInDialog` is `false`', () => {
+                // Arrange
+                const wrapper = shallowMount(VueCheckout, {
+                    store: createStore({
+                        ...defaultCheckoutState,
+                        isLoggedIn: true
+                    }),
+                    i18n,
+                    localVue,
+                    propsData,
+                    data () {
+                        return {
+                            nonFulfillableError: {
+                                code: ERROR_CODE_FULFILMENT_TIME_UNAVAILABLE,
+                                shouldShowInDialog: false
+                            }
+                        };
+                    }
+                });
+
+                // Act
+                const result = wrapper.vm.shouldShowErrorDialog;
+
+                // Assert
+                expect(result).toBe(false);
+            });
+
+            it('should return `false` if there is no error', () => {
+                // Arrange
+                const wrapper = shallowMount(VueCheckout, {
```

Comment:
Could the wrappers for these tests go in a beforeEach?

Reply:
Each usage has different data provided so thought I'd have to shallow mount it in each test to provide this different data? I know Enzyme with React lets you change state/props after mounting but didn't see any case of that inside our tests.

Reply:
👍  I only looked at the store part of it. Gonna blame Monday mornings